src/services/OParl/consultation.test.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  basicImportTest,
+  invalidUrlTest,
+  setupTestDataBase,
+} from '../../helpers';
+import { importConsultation } from './consultation';
+
+const helperFuncs = setupTestDataBase('test_consultation_import');
+
+// TODO: change test url to something within our control, that uses all fields
+const testUrl =
+  'https://www.muenchen-transparent.de/oparl/v1.0/consultation/48823';
+
+describe('importing a consultation', () => {
+  beforeAll(helperFuncs.beforeAll);
+
+  it('imports properly', basicImportTest(importConsultation, testUrl));
+
+  it(
+    'import returns undefined on faulty url',
+    invalidUrlTest(importConsultation),
+  );
+
+  afterAll(helperFuncs.afterAll);
+});
```

Comment:
some files are syntax highlighted by github but others like this here not, strange.

Reply:
for me it is highlighted
![Screenshot 2021-04-09 at 12 39 39](https://user-images.githubusercontent.com/59824597/114168534-acb5af00-9930-11eb-9112-1872644e6be2.png)

Reply:
i can say the same now.

<img width="589" alt="Bildschirmfoto 2021-04-09 um 13 13 27" src="https://user-images.githubusercontent.com/1942953/114172018-631b9300-9935-11eb-84cc-7924652288c5.png">

it was all black text before